CMakePresets.json

```diff
@@ -118,6 +118,14 @@
       "configurePreset": "host-gcc10",
       "jobs": 4,
       "targets": ["build-and-test"]
+    },
+    {
+      "name": "bootloader",
```

Comment:
this is unfortunately going to need an entry for the version that goes on the different pipette micro